ios/Classes/PdftronFlutterPlugin.m

```diff
@@ -1555,6 +1559,52 @@ - (void)importAnnotationCommand:(NSString *)xfdfCommand resultToken:(FlutterResu
     }
 }
 
+- (void)addAnnotations:(NSString *)annotations resultToken:(FlutterResult)flutterResult
```

Comment:
Since we only currently support FreeText, can we have an error message that states exactly why other types aren't possible (now)?